README.md

```diff
@@ -29,62 +30,49 @@ npm install @elastic/elasticsearch
 
 The minimum supported version of Node.js is `v8`.
 
-The library is compatible with all Elasticsearch versions since 5.x, but you should use the same major version of the Elasticsearch instance that you are using.
-```
-# Elasticsearch 7.x
-@elastic/elasticsearch@7
+The library is compatible with all Elasticsearch versions since 5.x, and you should use the same major version of the Elasticsearch instance that you are using.
 
-# Elasticsearch 6.x
-@elastic/elasticsearch@6
+| Elasticsearch Version | Client Version |
+| --------------------- |----------------|
+| `master`              | `master`       |
+| `7.x`                 | `7.x`          |
+| `6.x`                 | `6.x`          |
+| `5.x`                 | `5.x`          |
 
-# Elasticsearch 5.x
-@elastic/elasticsearch@5
+To install a specific major of the client, run the following command:
 ```
-
-#### Install multiple versions
-If you are using multiple versions of Elasticsearch, you need to use multiple versions of the client. In the past, install multiple versions of the same package was not possible, but with `npm v6.9`, you can do that via aliasing.
-
-The command you must run to install different version of the client is:
-```sh
-npm install <alias>@npm:@elastic/elasticsearch@<version>
-```
-So for example if you need to install `7.x` and `6.x`, you will run
-```sh
-npm install es6@npm:@elastic/elasticsearch@6
-npm install es7@npm:@elastic/elasticsearch@7
-```
-And your `package.json` will look like the following:
-```json
-"dependencies": {
-  "es6": "npm:@elastic/elasticsearch@^6.7.0",
-  "es7": "npm:@elastic/elasticsearch@^7.0.0"
-}
+npm install @elastic/elasticsearch@<major>
 ```
-And finally, you will require the packages from your code by using the alias you have defined.
-```js
-const { Client: Client6 } = require('es6')
-const { Client: Client7 } = require('es7')
 
-const client6 = new Client6({ node: 'http://localhost:9200' })
-const client7 = new Client7({ node: 'http://localhost:9201' })
+#### Browser
 
-client6.info(console.log)
-client7.info(console.log)
-```
+There is no official support for the browser environment since expose your Elasticsearch instance to everyone could lead to security issues.
```

Comment:
> **Warning**: There is no official support for the browser environment since expose your Elasticsearch instance to everyone could lead to security issues.